src/app/shared/components/pagination/pagination.component.scss

```diff
@@ -0,0 +1,21 @@
+.outerPagination {
```

Comment:
use kebap-case

Reply:
I have system with the classes. Because I'm using bootstrap and there the classes are kebap-case and they are reusable. So I'm continuing using this style for my custom classes witch can be used multiple times like grey-color, text-bold and etc, in every other case I'm using camel case.